### DISCUSSION TITLE
How to set a value to null in languages.toml?

### DISCUSSION THREAD

**Discussion by ameknite** [helix-editor/helix#7443] [Q&A]
I want to set a value to null to enable the unlimited maxLength feature  for the rust-analyzer config in my `languages.toml`

but I cannot set it to null, I tried {} , 0, "", but nothing works.
```
[language.config.inlayHints]
maxLength = null
```

The rust analyzer manual tell this, but only works in json, how I do the same in toml?
```
rust-analyzer.inlayHints.maxLength (default: 25)

Maximum length for inlay hints. Set to null to have an unlimited length.
```

**Comment by the-mikedavis** [comment]
It looks like TOML doesn't support null values: https://github.com/toml-lang/toml/issues/30. As a workaround you could configure that setting to a very large number.
